autoload/intero/util.vim

```diff
@@ -79,4 +79,22 @@ function! intero#util#get_selection(l1, c1, l2, c2) abort
     return join(l:lines, "\n")
 endfunction
 
+function! intero#util#strip_control_characters(line) abort
+    " Filter out ANSI codes - they are needed for interactive use, but we
+    " don't care about them. Note that we replace [0-?] with [0-Z] to filter
+    " out the arrow keys as well (xterm codes)
+    "
+    " https://stackoverflow.com/questions/14693701/
+    let s:regex1 = '\([\x9b]\|[\x1b]\[\)[0-Z]*[ -\/]*[@-~]'
```

Comment:
Can we use [`\v` (very magic)](http://vimdoc.sourceforge.net/htmldoc/pattern.html#/magic) to improve the readability a bit?